devel/bazelisk/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/bazelbuild/bazelisk 1.11.0 v
+revision            0
+categories          devel
+maintainers         {@0xIDANT danieltrautmann.com:me} openmaintainer
+license             Apache-2
+
+description         A user-friendly launcher for Bazel
+
+long_description    Bazelisk is a wrapper for Bazel written in Go. \
+                    It automatically picks a good version of Bazel given your \
+                    current working directory, downloads it from the official \
+                    server (if required) and then transparently passes through \
+                    all command-line arguments to the real Bazel binary. You \
+                    can call it just like you would call Bazel.
+
+conflicts           bazel
+
+build.cmd           ./build.sh
+build.env-append    BAZELISK_VERSION=v${version}
+build.env-append    GOPROXY=https://proxy.golang.org,direct
```

Comment:
> I'm not quite sure if it is a good idea to set the GOPROXY environment variable back to it's default value.

That's fine, plenty of Go ports do so already.  But instead of this, use:
```
build.env-delete    GO111MODULE=off GOPROXY=off
```

...to simply remove the environment variables that the `golang` portgroup sets by defalt.

Reply:
Thanks for the hint